scripts/lib/build-utils.mjs

```diff
@@ -224,17 +225,33 @@ export default {
    * @param os {"windows" | "linux"}
    */
   async buildNerdctlStub(os) {
-    const platDir = os === 'windows' ? 'win32' : os === 'linux' ? 'linux' : '<unknown>';
-    const basename = os === 'windows' ? 'nerdctl.exe' : 'nerdctl-stub';
-    const outFile = path.join(this.srcDir, 'resources', platDir, 'bin', basename);
+    if (!['windows', 'linux'].includes(os)) {
+      throw new Error(`Unexpected os of ${ os }`);
+    }
+    let platDir, basename, parentDir, outFile, sourceFile, destFile;
 
+    if (os === 'windows') {
+      platDir = 'win32';
+      parentDir = path.join(this.srcDir, 'resources', platDir, 'bin');
+      outFile = path.join(parentDir, 'nerdctl.exe');
+      sourceFile = outFile = path.join(parentDir, 'nerdctl.exe');
+      destFile = path.join(parentDir, 'docker.exe');
+    } else {
+      platDir = 'linux';
+      parentDir = path.join(this.srcDir, 'resources', platDir, 'bin');
+      outFile = path.join(parentDir, 'nerdctl-stub');
+      sourceFile = path.join(parentDir, 'nerdctl');
```

Comment:
We should probably have a comment on how `nerdctl` is a script, and _not_ the just-built go binary.

Reply:
done